src/Playroom/Frame.tsx

```diff
@@ -19,11 +19,15 @@ export default function Frame({
   components,
   FrameComponent,
 }: FrameProps) {
-  const { themeName, code } = useParams((rawParams) => ({
-    themeName:
-      typeof rawParams.themeName === 'string' ? rawParams.themeName : '',
-    code: typeof rawParams.code === 'string' ? rawParams.code : '',
-  }));
+  const { themeName, code } = useParams((rawParams) => {
+    const rawThemeName = rawParams.get('themeName');
+    const rawCode = rawParams.get('code');
+
+    return {
+      themeName: rawThemeName || '',
+      code: rawCode || '',
+    };
+  });
```

Comment:
`query-string`'s API could return an array, but [`URLSearchParams` `.get` API](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get) returns `string | null`, so there's no need for a `typeof` check here anymore.